internal/fsutil/path_test.go

```diff
@@ -0,0 +1,52 @@
+package fsutil_test
+
+import (
+	. "gopkg.in/check.v1"
+
+	"github.com/canonical/chisel/internal/fsutil"
+)
+
+var dirTests = []struct {
```

Comment:
Could `dirTests` and `cleanTests` be declared inside the test rather than globally?

Reply:
This would deviate from how it's currently done in Chisel since its inception before I even started contributing. I'll let the maintainer decide on this one.